tests/integration/preprocessor/_regrid/test_regrid.py

```diff
@@ -35,9 +35,69 @@ def test_regrid__linear(self):
                                     coord_system=self.cs)
         coords_spec = [(lats, 0), (lons, 1)]
         grid = iris.cube.Cube(data, dim_coords_and_dims=coords_spec)
-        result = regrid(self.cube, grid, 'linear')
+        self.grid_for_linear = grid
+
+        # Setup unstructed cube and grid
+        data = np.empty((1, 1))
+        lons = iris.coords.DimCoord([1.6],
+                                    standard_name='longitude',
+                                    bounds=[[1, 2]],
+                                    units='degrees_east',
+                                    coord_system=self.cs)
+        lats = iris.coords.DimCoord([1.6],
+                                    standard_name='latitude',
+                                    bounds=[[1, 2]],
+                                    units='degrees_north',
+                                    coord_system=self.cs)
+        coords_spec = [(lats, 0), (lons, 1)]
+        self.grid_for_unstructured_nearest = iris.cube.Cube(
+            data, dim_coords_and_dims=coords_spec)
+
+        # Replace 1d spatial coords with 2d spatial coords.
+        lons = self.cube.coord('longitude')
+        lats = self.cube.coord('latitude')
+        x, y = np.meshgrid(lons.points, lats.points)
+
+        lats = iris.coords.AuxCoord(
+            x,
```

Comment:
lats should be y and lons x, no?

Reply:
That was copy-pasted from existing code, and I really do not want to mess with other tests

Reply:
Your code, your responsibility. The issue is that we have a few lines before this
```
x, y = np.meshgrid(lons.points, lats.points)
```
clearly saying that x is longitude and y is latitudes (which makes sense). If the switch is purposeful, it should be explained in a comment, otherwise, it should be corrected.